packages/ckeditor5-paste-from-office/src/filters/space.js

```diff
@@ -39,9 +39,13 @@ export function normalizeSpacing( htmlString ) {
  */
 export function normalizeSpacerunSpans( htmlDocument ) {
 	htmlDocument.querySelectorAll( 'span[style*=spacerun]' ).forEach( el => {
-		const innerTextLength = el.innerText.length || 0;
+		// For paste contents from WPS Office, the span's inner text is not all of spaces or blank,
+		// so we should keep the origin contents, or replace ONLY space characters.
+		if ( /[^\b]/.test( el.innerText.trim() ) === false ) {
```

Comment:
What's the purpose of the `[^\b]` check here? What are we actually checking here? Couldn't it be replaced with `el.innerText.trim().length === 0`?

Reply:
Thanks for your reply. I will focus on this during the holidays. Please forgive my delay.